### ISSUE TITLE
Automatically change count limit based on the color of the send button

### ISSUE THREAD

**Issue by JukMR** [qequ/chatgpt_counter#4]
We can automate the process of switching between GPT 3.5 and GPT 4 word limit by checking what the color of the button is.

**Comment by qequ** [comment]
Need to be adressed if the color changes due to the user using a dark reader mode or alike

**Comment by JukMR** [comment]
Yes! Having that in mind i propose to check in the url for what is the current model selected or if not, we could check the conversation details in the top of the webpage:

![image](https://github.com/qequ/chatgpt_counter/assets/26848541/ef959301-dfe3-48fb-9090-e0ccd6a47df1)
![image](https://github.com/qequ/chatgpt_counter/assets/26848541/ba2ebdf4-2194-4888-8812-650e1c33b674)

**Comment by qequ** [comment]
the url of each chat changes with a custom hash, I don't think it's possible that way. The info tab seems a better approach

**Comment by JukMR** [comment]
Also, as another alternative, the chat image does change color depending if the conversation is gpt 3.5 or 4:

![image](https://github.com/qequ/chatgpt_counter/assets/26848541/2ba21e27-efae-4fce-b5af-d7cfdc48f5ca)
![image](https://github.com/qequ/chatgpt_counter/assets/26848541/b9fcf45b-b48f-4114-95f4-0b2ab4a46c21)

This could be another approach
